handler/handler.go

```diff
@@ -235,12 +236,12 @@ type apqAdapter struct {
 	PersistedQueryCache
 }
 
-func (a apqAdapter) Get(ctx context.Context, key string) (value any, ok bool) {
+func (a apqAdapter) Get(ctx context.Context, key string) (value string, ok bool) {
 	return a.PersistedQueryCache.Get(ctx, key)
 }
 
-func (a apqAdapter) Add(ctx context.Context, key string, value any) {
-	a.PersistedQueryCache.Add(ctx, key, value.(string))
+func (a apqAdapter) Add(ctx context.Context, key string, value string) {
```

Comment:
If you could satisfy the linter here
```suggestion
func (a apqAdapter) Add(ctx context.Context, key, value string) {
```